tools/pipelines/templates/upload-json-steps.yml

```diff
@@ -19,24 +19,87 @@ steps:
   clean: true
 
 # Download the api-extractor outputs
-- download: common_definitions
-  artifact: _api-extractor-temp
-- download: common_utils
-  artifact: _api-extractor-temp
-- download: container_definitions
-  artifact: _api-extractor-temp
-- download: core_interfaces
-  artifact: _api-extractor-temp
-- download: driver_definitions
-  artifact: _api-extractor-temp
-- download: protocol_definitions
-  artifact: _api-extractor-temp
-- download: azure
-  artifact: _api-extractor-temp
-- download: client
-  artifact: _api-extractor-temp
-- download: server
-  artifact: _api-extractor-temp
+- task: DownloadPipelineArtifact@2
```

Comment:
Nit: It might be worth introducing a tiny template that wraps this task call and takes in the necessary parameters. Would make the calls here much shorter and easier to read.